src/Engine/AlgoliaEngine.php

```diff
@@ -93,6 +93,10 @@ private function batchUpdate($searchableEntities)
     {
         $data = [];
         foreach ($searchableEntities as $entity) {
+            if (empty($entity->getSearchableArray())) {
+                break;
```

Comment:
Looking at this again, I think this should be `continue`, not `break`. We want to skip the current entity but keep looping. What do you think?